### PR TITLE
eWeLink / Sonoff Motion sensor no need occupancy timeout

### PR DESCRIPTION
**Pull request by kirovilya**

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -9936,7 +9936,7 @@ const devices = [
         vendor: 'eWeLink',
```
Comment (Koenkk) [inline devices.js:9936]: Question, are these the same as the sonoff devices? (https://sonoff.tech/product/smart-home-security/snzb-03). If so I think we should mark these all as Sonoff (I can do this after this PR).

Comment (kirovilya) [reply devices.js:9936]: yes. modelID is the same for all devices

Comment (kirovilya) [reply devices.js:9936]: but outside they look some different :)

Comment (Koenkk) [reply devices.js:9936]: Would it make sense to mark them as Sonoff and add Ewelink as the whitelabel? (as sonoff is more common?)

**Pull request merged**